src/comment.ts

```diff
@@ -196,7 +196,7 @@ export const comment = async () => {
         }),
       ] as const)
 
-    const addChangesetUrl = `${CI_MERGE_REQUEST_PROJECT_URL!}/new/${mrBranch}?file_name=.changeset/${humanId(
+    const addChangesetUrl = `${CI_MERGE_REQUEST_PROJECT_URL!}/-/new/${mrBranch}?file_name=.changeset/${humanId(
```

Comment:
Any reference document? They two both work on our internal GitLab instance.

Reply:
Considering it **is** a real problem from you and the link url works for me at the same time, I'll merge this PR.